src/nni_manager/training_service/reusable/environments/openPaiEnvironmentService.ts

```diff
@@ -248,19 +244,6 @@ export class OpenPaiEnvironmentService extends EnvironmentService {
         return deferred.promise;
     }
 
-    private async refreshPlatform(): Promise<void> {
```

Comment:
why remove the refresh logic?

Reply:
It's ok to remove, as it only refresh pai token. And it's a pattern for future platform, which need regular refresh credential.

Reply:
@squirrelsc  you mean we will add dedicated token refresh logic in the future?

Reply:
I mean, if there are similar requirements, here is a reference.

Reply:
then, why remove pai token refresh? no need to refresh token any more on pai?